src/stream/src/executor/backfill/snapshot_backfill/executor.rs

```diff
@@ -151,6 +152,8 @@ impl<S: StateStore> SnapshotBackfillExecutor<S> {
             }
         };
         let first_recv_barrier_epoch = first_recv_barrier.epoch;
+        let backfill_paused =
```

Comment:
May rename to `initial_backfill_paused`. It's confusing that the `backfill_paused` is not a mutable variable.

Reply:
https://github.com/risingwavelabs/risingwave/pull/22215/commits/3432711be429a9e32806f60a6f52f1091c7b0b72